.github/workflows/CodeQuality.yaml

```
@@ -0,0 +1,41 @@
+on:
+  pull_request:
+    types: [synchronize, opened, reopened]
+
+jobs:
+  unit-test:
+    runs-on: ubuntu-latest
+    container:
+      image: node:22
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)